h2o-py/tests/testdir_jira/pyunit_pubdev_4723.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import h2o
+from tests import pyunit_utils
+
+
+def test_4723():
+    frame = h2o.create_frame()
```

Comment:
The test should also show that the content of the frame is correct.

What if "NA" constant is found in the source data? Eg. "NA" as in North America. What does the parser do?